app/src/main/java/com/coderGtm/yantra/terminal/Terminal.kt

```diff
@@ -65,7 +65,7 @@ class Terminal(
     private val getSecondarySuggestions = preferenceObject.getBoolean("getSecondarySuggestions",true)
     
     private var commandQueue: MutableList<String> = mutableListOf()
```

Comment:
please add this public declaration along with other public vars, possibly after workingDir